include/evmc/evmc.h

```diff
@@ -974,12 +974,12 @@ struct evmc_instance
     evmc_execute_fn execute;
 
     /**
-     * Pointer to function returning capabilities supported by the VM instance.
+     * A method returning capabilities supported by the VM instance.
      *
-     * The value returned might change when different options are requested via set_option.
+     * The value returned MAY change when different options are set via the set_option() method.
```

Comment:
Is there a way we could add a test for this? Does it make sense to add a test for this?

Reply:
We could add behavior like that to example_vm but there is nothing to test here about conformance. Anyway, I'm going to remove this "feature" in EVMC 7.